### PR TITLE
add improvement.py

### PR DESCRIPTION
**Pull request by huitang-earth**
test pull request

### CODE REVIEW

**Review by ejgal — APPROVED** [review]
Looks good to me @huitang-earth!

File: src/improvement.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+for num_measurements in [25, 100, 500]:
+
+   temperatures = read_data(
+       file_name="data/temperatures.csv",
+       nrows=num_measurements,
+       column="Air temperature (degC)",
+   )
+
+   mean = np.mean(temperatures)
```
Comment (irismuz) [inline src/improvement.py:26]: Thank you for this effort. Could you please let me know which library did you use here?

**Review by irismuz — COMMENTED** [review]
Thank you for this pull request. Could you please check my comment?

File: src/improvement.py (hunk 1 above)
Comment (huitang-earth) [reply src/improvement.py:26]: It uses "numpy" library.  @irismuz

**Review by irismuz — APPROVED** [review]

**Pull request closed without merging**